vendor/github.com/OpenBazaar/spvwallet/headers.go

```
@@ -63,11 +74,15 @@ var (
 )
 
 func NewHeaderDB(filePath string) *HeaderDB {
+	if !strings.Contains(filePath, ".bin") {
+		filePath = path.Join(filePath, "headers.bin")
+	}
 	h := new(HeaderDB)
-	db, _ := bolt.Open(path.Join(filePath, "headers.bin"), 0644, &bolt.Options{InitialMmapSize: 5000000})
+	db, _ := bolt.Open(filePath, 0644, &bolt.Options{InitialMmapSize: 5000000})
```

If this error is ignored it will panic on line 87